Source/Management.GraphQL/Tenancy/Tenants.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Dolittle.Runtime.Tenancy;
+
+namespace Dolittle.Runtime.Management.GraphQL.Tenancy
+{
+    public class Tenants
+    {
+        readonly ITenants _tenants;
+
+        public Tenants(ITenants tenants)
+        {
+            _tenants = tenants;
+        }
+
+        public IEnumerable<Guid> All() => _tenants.All.Select(_ => _.Value);
```

Comment:
Is there no easy way to convert ConceptOf things in HotChocolate?

Reply:
Yup. And I’ve done all of this in Vanir - but I didn’t feel like duplicating that logic just yet.